app/assets/stylesheets/components/stories.scss

```diff
@@ -13,8 +13,9 @@
   background: var(--card-bg);
   font-size: var(--fs-base); // Todo: remove when ready.
   box-shadow: 0 0 0 1px var(--card-border);
-  margin-bottom: var(--su-2);
+  margin: 0 var(--su-3) var(--su-3);
```

Comment:
Refactor (nitpick): Visually things looks great. I think it may make more sense to set the story cards parent to a grid and give it a grid gap instead so that the card does not need to worry about margins.

Reply:
@nickytonline I love that idea and began investigating how we might do this. Hmm, upon further investigation, I'm not sure it will be possible to use a grid-gap on the parent container for all of the `crayons-story` because that parent `div` seems to be automatically-generated. Let me illustrate the rendered code in the browser:

![Image 2021-05-28 at 9 18 55 AM](https://user-images.githubusercontent.com/816402/120013827-1ef05a80-bf96-11eb-8a64-90de4db8a101.jpg)

Given this scenario, what are you suggestions you have? Shall we keep the approach we have right now, or invest further work into the refactoring using a grid-gap?

Reply:
@lisasy, the `<div />` is generated on line 93 in the hompageFeed.jsx file

https://github.com/forem/forem/blob/ac898238c1250b0961af1da65a7a2afa8c20764c/app/javascript/packs/homePageFeed.jsx#L89-L94

The one thing we'd need to check is when the infinite scroll kicks in as well. This is what that looks like this

```
<div class="loading-articles" id="loading-articles" style="display: none;">
	loading...
</div>
```

That exists in a few files.

![image](https://user-images.githubusercontent.com/833231/120030099-03e90f00-bfc5-11eb-9fb7-878496e78311.png)

If this is too much to take on right now, we can do it in a subsequent PR.

Reply:
@nickytonline thanks Nick — let's do that refactoring/optimization in a separate PR — we'll discuss this tomorrow in our 1:1 :)